constants.bal

```diff
@@ -0,0 +1,254 @@
+const string SETTING_BOTH_VALUES_ERROR = "Cannot set both throughput and maxThroughput headers at once";
+const string NULL_PARTITIONKEY_VALUE_ERROR = "Partition key values are null";
+const string INDEXING_DIRECTIVE_ERROR = "Indexing directive should be either Exclude or Include";
+const string CONSISTANCY_LEVEL_ERROR = "Consistency level should be one of Strong, Bounded, Session, or Eventual";
+const string VALIDITY_PERIOD_ERROR = "Resource token validity period must be between 3600 and 18000";
+const string MASTER_KEY_ERROR = "Enter a valid master key and token type should be master key";
+const string JSON_PAYLOAD_ACCESS_ERROR = "Error occurred while accessing the JSON payload of the response";
+const string REST_API_INVOKING_ERROR = "Error occurred while invoking the REST API";
+const string NULL_RESOURCE_TYPE_ERROR = "ResourceType is incorrect/null";
+const string NULL_DATE_ERROR = "Date is invalid/null";
+const string NULL_AUTHORIZATION_SIGNATURE_ERROR = "Authorization token is null";
+const string DECODING_ERROR = "Base64 Decoding error";
+const string TIME_STRING_ERROR = "Time string is not correct";
+const string INVALID_RESPONSE_PAYLOAD_ERROR = "Invalid response payload";
+const string STREAM_IS_NOT_TYPE_ERROR = "The stream is not type";
+const string INVALID_STREAM_TYPE = "Invalid stream type";
+const string AZURE_ERROR = "Error occured";
+const string PAYLOAD_IS_NOT_JSON_ERROR = "Request payload is not json";
+const string EMPTY_BASE_URL_ERROR = "Base URL cannot be empty";
+const string EMPTY_MASTER_TOKEN_ERROR = "Master token cannot be empty";
+const string INVALID_MASTER_TOKEN_ERROR = "Master token is not valid";
+
+# JSON keys in response
+const string JSON_KEY_ID = "id";
+const string JSON_KEY_RESOURCE_ID = "_rid";
+const string JSON_KEY_SELF_REFERENCE = "_self";
+const string JSON_KEY_ETAG = "_etag";
+const string JSON_KEY_TIMESTAMP = "_ts";
+const string JSON_KEY_ATTACHMENTS = "_attachments";
+const string JSON_KEY_DOCUMENTS = "Documents";
+const string JSON_KEY_OFFERS = "Offers";
+
+# Fields in record types
+const string JSON_KEY_PERMISSIONMODE = "permissionMode";
+const string JSON_KEY_RESOURCE = "resource";
+const string JSON_KEY_OFFER_RESOURCE_ID = "offerResourceId";
+const string JSON_KEY_OFFER_TYPE = "offerType";
+const string JSON_KEY_OFFER_VERSION = "offerVersion";
+const string JSON_KEY_CONTENT = "content";
+
+# Property in record type representing headers
+const string RESPONSE_HEADERS = "reponseMetadata";
+
+# Elements in an error response
+const string ACTIVITY_ID = "ActivityId";
+const string STATUS = "status";
+const string STATUS_NOT_FOUND_STRING = "404";
+
+# Numeric constants
+const int MIN_REQUEST_UNITS = 400;
+const int MIN_TIME_TO_LIVE_IN_SECONDS = 3600;
+const int MAX_TIME_TO_LIVE_IN_SECONDS = 18000;
+
+# Algorithm Used for partitioning
+const string PARTITIONING_ALGORITHM_TYPE_HASH = "Hash";
+const int DEFAULT_PARTITION_KEY_VERSION = 1;
```

Comment:
IMO, if we rename this to `PARTITION_KEY_VERSION_1` it would be more meaningful and intuitive.

Reply:
Renamed this constant